cli/packages/util/credentials.go

```diff
@@ -87,6 +88,13 @@ func GetCurrentLoggedInUserDetails() (LoggedInUserDetails, error) {
 			SetAuthToken(userCreds.JTWToken).
 			SetHeader("Accept", "application/json")
 
+		config.INFISICAL_URL_MANUAL_OVERRIDE = config.INFISICAL_URL
```

Comment:
Can you explain why you choose to use `INFISICAL_URL_MANUAL_OVERRIDE` variable instead of   `config.INFISICAL_URL`? I see in `GetCurrentLoggedInUserDetails`, you set `INFISICAL_URL_MANUAL_OVERRIDE=config.INFISICAL_URL` so i'm curious if we need this or if there is a use case I didn't catch